airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/CdcPostgresSourceTest.java

```diff
@@ -88,8 +88,10 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(CdcPostgresSourceTest.class);
 
   private static final String SLOT_NAME_BASE = "debezium_slot";
-  private static final String MAKES_STREAM_NAME = "public.makes";
-  private static final String MODELS_STREAM_NAME = "public.models";
+  private static final String MAKES_SCHEMA = "public";
```

Comment:
Modified one of the tables to be in a separate schema to confirm data is propagated correctly. I think this is correct, but please sanity check my work.